Cython/Utility/ObjectHandling.c

```diff
@@ -1421,10 +1438,10 @@ static CYTHON_INLINE PyObject *__Pyx__GetModuleGlobalName(PyObject *name)
         return result;
     }
 #else
-    result = PyDict_GetItem($moddict_cname, name);
+    __Pyx_PyDict_GetItemRef(result, $moddict_cname, name);
```

Comment:
```suggestion
    if (unlikely(__Pyx_PyDict_GetItemRef(result, $moddict_cname, name) == -1)) PyErr_Clear();
```

It looks like this function is just ignoring errors so I'll keep doing that here.